eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -36,11 +36,11 @@ function CreateReleases($pkgList, $releaseApiUrl, $releaseSha) {
       "Authorization" = "token $($env:GH_TOKEN)"
     }
 
-    Invoke-WebRequest-WithHandling -url $url -body $body -headers $headers -method "Post"
+    Invoke-RestMethod-WithHandling -url $url -body $body -headers $headers -method "Post"
   }
 }
 
-function Invoke-WebRequest-WithHandling($url, $method, $body = $null, $headers = $null) {
+function Invoke-RestMethod-WithHandling($url, $method, $body = $null, $headers = $null) {
```

Comment:
As we chatted it would be good to just remove this method completely. The extra data about rate-limiting isn't all that interesting to keep this helper around.